api-gateway/src/main/java/com/petclinic/bffapigateway/dtos/Vets/PhotoDetails.java

```diff
@@ -11,7 +11,6 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class PhotoDetails {
-
     private int id;
```

Comment:
That is not what SQL injection is...

Reply:
I don't use PhotoDetails. I think it was Customer or Bill that used it

Reply:
> That is not what SQL injection is...

I know, sorry I should've explained myself better, I meant that if you had the exact id of the image in the database, it could lead to data leak since it's less secure.

Reply:
> I don't use PhotoDetails. I think it was Customer or Bill that used it

Ooh I see, thank you for letting me know.